script.js

```diff
@@ -1,12 +1,77 @@
 //You can edit ALL of the code here
+
 function setup() {
   const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
 }
-
+//Level 100
 function makePageForEpisodes(episodeList) {
   const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  rootElem.innerHTML = "";
+  episodeList.forEach((element) => {
+    //creating DOM elements
+    let divElement = document.createElement("div");
+    let headerElement = document.createElement("h3");
+    let imgElement = document.createElement("img");
+    let pElement = document.createElement("p");
+    let aElement = document.createElement("a");
```

Comment:
Suggestion: Group up logical sections of your code to make it easier to read, so instead of creating all of your elements at the start of the function create 1 element and add the appropriate properties to it then leave a line and create a second element. ect.